### ISSUE TITLE
@throws not used when on new line

### ISSUE THREAD

**Issue by marcwrobel** [stoicflame/enunciate#64]
Javadoc @throws description is not used when on new line :
```
/**
 * ...
 * @throws InvalidParameterException
 *         if the given parameter is invalid
 */
```

It works if on the same line :
```
/**
 * ...
 * @throws InvalidParameterException if the given parameter is invalid
 */
```

**Comment by marcwrobel** [comment]
This issue was previously a codehaus Jira issue : https://jira.codehaus.org/browse/ENUNCIATE-751.

**Comment by marcwrobel** [comment]
In the previous ticket the bug was confirmed and located down in the underlying APT engine template processor, which is deprecated.

This bug will not be fixed until the move to javac (https://jira.codehaus.org/browse/ENUNCIATE-701).

**Comment by stoicflame** [comment]
Thanks for the report.

**Comment by stoicflame** [comment]
Assigning this issue to v2.1.0.

**Comment by stoicflame** [comment]
Fixed at 3e883d3 for 2.1.

**Comment by stoicflame** [comment]
[Enunciate 2.1 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.1.0)
